audit/projects/k8s-infra-prow-build/services/monitoring/dashboards/build-cluster-capacity-usage.json

```diff
@@ -0,0 +1 @@
ts":[{"minAlignmentPeriod":"60s","plotType":"LINE","timeSeriesQuery":{"timeSeriesFilter":{"aggregation":{"alignmentPeriod":"60s","crossSeriesReducer":"REDUCE_COUNT","groupByFields":["metadata.user_labels.\"prow.k8s.io/type\""],"perSeriesAligner":"ALIGN_RATE"},"filter":"metric.type=\"kubernetes.io/pod/network/received_bytes_count\" resource.type=\"k8s_pod\" metadata.system_labels.\"node_name\"=monitoring.regex.full_match(\"gke-prow-build-pool.*\") resource.label.\"namespace_name\"=\"test-pods\""}}}],"timeshiftDuration":"0s","yAxis":{"label":"y1Axis","scale":"LINEAR"}}},{"title":"Pods in test-pods NS per node (stats mode)","xyChart":{"chartOptions":{"mode":"STATS"},"dataSets":[{"minAlignmentPeriod":"60s","plotType":"LINE","timeSeriesQuery":{"timeSeriesFilter":{"aggregation":{"alignmentPeriod":"60s","crossSeriesReducer":"REDUCE_COUNT","groupByFields":["metadata.system_labels.\"node_name\""],"perSeriesAligner":"ALIGN_RATE"},"filter":"metric.type=\"kubernetes.io/pod/network/received_bytes_count\" resource.type=\"k8s_pod\" metadata.system_labels.\"node_name\"=monitoring.regex.full_match(\"gke-prow-build-pool.*\") resource.label.\"namespace_name\"=\"test-pods\""}}}],"timeshiftDuration":"0s","yAxis":{"label":"y1Axis","scale":"LINEAR"}}},{"title":"Pods in test-pods NS by org/ref#pull","xyChart":{"chartOptions":{"mode":"COLOR"},"dataSets":[{"legendTemplate":"${metadata.user_labels\\.prow\\.k8s\\.io/refs\\.org}/${metadata.user_labels\\.prow\\.k8s\\.io/refs\\.repo}#${metadata.user_labels\\.prow\\.k8s\\.io/refs\\.pull}","minAlignmentPeriod":"60s","plotType":"LINE","timeSeriesQuery":{"timeSeriesFilter":{"aggregation":{"alignmentPeriod":"60s","crossSeriesReducer":"REDUCE_COUNT","groupByFields":["metadata.user_labels.\"prow.k8s.io/refs.pull\"","metadata.user_labels.\"prow.k8s.io/refs.repo\"","metadata.user_labels.\"prow.k8s.io/refs.org\""],"perSeriesAligner":"ALIGN_RATE"},"filter":"metric.type=\"kubernetes.io/pod/network/received_bytes_count\" resource.type=\"k8s_pod\" metadata.system_labels.\"node_name\"=monitoring.regex.full_match(\"gke-prow-build-pool.*\") resource.label.\"namespace_name\"=\"test-pods\" metadata.user_labels.\"prow.k8s.io/type\"=\"presubmit\""}}}],"timeshiftDuration":"0s","yAxis":{"label":"y1Axis","scale":"LINEAR"}}},{"title":"Pods in test-pods NS by prow.k8s.io/job","xyChart":{"chartOptions":{"mode":"COLOR"},"dataSets":[{"minAlignmentPeriod":"60s","plotType":"LINE","timeSeriesQuery":{"timeSeriesFilter":{"aggregation":{"alignmentPeriod":"60s","crossSeriesReducer":"REDUCE_COUNT","groupByFields":["metadata.user_labels.\"prow.k8s.io/job\""],"perSeriesAligner":"ALIGN_RATE"},"filter":"metric.type=\"kubernetes.io/pod/network/received_bytes_count\" resource.type=\"k8s_pod\" metadata.system_labels.\"node_name\"=monitoring.regex.full_match(\"gke-prow-build-pool.*\") resource.label.\"namespace_name\"=\"test-pods\""}}}],"timeshiftDuration":"0s","yAxis":{"label":"y1Axis","scale":"LINEAR"}}}]},"name":"projects/773781448124/dashboards/10925237040785467832"}
```

Comment:
This was me manually renaming the dashboard in the UI; we're not yet managing these via automation (ref: https://github.com/kubernetes/k8s.io/issues/1376)